<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostDataUi</class>
 <widget class="QWidget" name="PostDataUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0,3">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="postTypeLabel">
       <property name="text">
        <string>Post Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="postTypeText">
         <property name="text">
          <string>Text</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">postTypeButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="postTypeLink">
         <property name="text">
          <string>Link</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">postTypeButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="postTypeImage">
         <property name="text">
          <string>Image</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">postTypeButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="postMessageLabel">
       <property name="text">
        <string>Message</string>
       </property>
       <property name="buddy">
        <cstring>postMessageText</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="postMessageText">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="postNameLabel">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>postNameText</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="postNameText"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="postLinkLabel">
       <property name="text">
        <string>Link</string>
       </property>
       <property name="buddy">
        <cstring>postLinkText</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="postLinkText"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="postPictureLabel">
       <property name="text">
        <string>Picture</string>
       </property>
       <property name="buddy">
        <cstring>postPictureText</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="postPictureText"/>
       </item>
       <item>
        <widget class="QToolButton" name="postPictureChooserButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="postTargetsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabGroups">
      <attribute name="title">
       <string>Groups</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QListWidget" name="groupsListWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="totalGroupsLabel">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Total Groups : 0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="groupsFilterText"/>
         </item>
         <item>
          <widget class="QCheckBox" name="selectAllGroupsCheckBox">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPages">
      <attribute name="title">
       <string>Pages</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QListWidget" name="pagesListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="totalPagesLabel">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Total Pages : 0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="pagesFilterText"/>
         </item>
         <item>
          <widget class="QCheckBox" name="selectAllPagesCheckBox">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLikes">
      <attribute name="title">
       <string>Likes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QListWidget" name="likesListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="totalLikesLabel">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Total Likes : 0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="likesFilterText"/>
         </item>
         <item>
          <widget class="QCheckBox" name="selectAllLikesCheckBox">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>postTypeButtonGroup</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>PostDataUi</receiver>
   <slot>postTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>postPictureChooserButton</sender>
   <signal>clicked()</signal>
   <receiver>PostDataUi</receiver>
   <slot>choosePostPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupsFilterText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostDataUi</receiver>
   <slot>filterGroups(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pagesFilterText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostDataUi</receiver>
   <slot>filterPages(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>likesFilterText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PostDataUi</receiver>
   <slot>filterLikes(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAllGroupsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PostDataUi</receiver>
   <slot>selectAllVisibleGroups(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAllPagesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PostDataUi</receiver>
   <slot>selectAllVisiblePages(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAllLikesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PostDataUi</receiver>
   <slot>selectAllVisibleLikes(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>postTypeChanged(int)</slot>
  <slot>choosePostPicture()</slot>
  <slot>filterGroups(QString)</slot>
  <slot>filterPages(QString)</slot>
  <slot>filterLikes(QString)</slot>
  <slot>selectAllVisibleGroups(bool)</slot>
  <slot>selectAllVisiblePages(bool)</slot>
  <slot>selectAllVisibleLikes(bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="postTypeButtonGroup"/>
 </buttongroups>
</ui>
